frontend/src/components/ExperimentCollection/ExperimentCollection.scss

```diff
@@ -220,7 +220,14 @@
                         padding: 0 15px 15px;
                         opacity: 0.8;
                     }
+                    .counter {
+                        color: white;
+                        position: relative;
```

Comment:
Interesting, I would expect the counter class to have `position: absolute;` and the experiment tile to have `position: relative;`. 🤔  But if it works, it works.